tests/core/Jalasoft.TeamUp.Projects.Core.Tests/Validators/ProjectValidatorTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Tests.Validators
+{
+    using System;
+    using FluentValidation.TestHelper;
+    using Jalasoft.TeamUp.Projects.Core.Validators;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Xunit;
+
+    public class ProjectValidatorTests
+    {
+        private readonly ProjectValidator validator;
+
+        public ProjectValidatorTests()
+        {
+            this.validator = new ProjectValidator();
+        }
+
+        [Fact]
+        public void Should_Have_Pass_Id_is_Valid()
+        {
+            // Arrange
+            var project = new Project { Name = "TeamUp", Id = Guid.NewGuid() };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldNotHaveValidationErrorFor(project => project.Id);
+        }
+
+        [Fact]
```

Comment:
please use [Theory] and group all the unit tests related to a property instead of creating a new one

Reply:
Its done